src/monobase/pget.py

```diff
@@ -123,6 +124,22 @@ def single_pget(url: str, dest: str, extract: bool, force: bool) -> None:
     if not force:
         assert not os.path.exists(dest)
 
+    m = hashlib.sha256()
+    m.update(url.encode())  # default encoding of utf-8
+    fpath = os.path.join(KNOWN_WEIGHTS_DIR, m.hexdigest())
+    if os.path.exists(fpath):
+        if extract:
+            # dest is a directory
+            os.makedirs(dest, exist_ok=True)
```

Comment:
`pget -x` fails if target exists. So we should do the `if force and os.path.exists(dest): shutil.rmtree(dest)` similar to the `else:` branch.